.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I haven't used `v1` yet, every time I've used this it's `uses: actions/checkout@v2` as in `test.yml`! Might as well use a newer version? (if that's what this means).

Reply:
This could be updated, but i just used what was already there in the core repo.